README.md

```diff
@@ -1,5 +1,4 @@
-# Brave Browser
-
+![Brave Browser](./docs/source/_static/Brave.svg)
```

Comment:
Your design looks great!

Reply:
Thank you for liking it! I'm eagerly waiting for the merge.

Reply:
that works! Thank you.